Cubical/Experiments/ZCohomologyExperiments.agda

```diff
@@ -1,79 +0,0 @@
-{-# OPTIONS --safe #-}
```

Comment:
@aljungstrom   Should this be kept or deleted?

Reply:
This one does not check.

Reply:
Commented out whole file.

Reply:
@aljungstrom Can't we just delete this then?